Vision/README.md

```diff
@@ -11,34 +9,34 @@ Our vision is for a World Wide Web that is more inclusive, and more respectful o
 
-Our new vision is built on these twin pillars:
-
-* Our existing pillar, on which we built the web: the enormous potential of technological change.
-* Our new pillar for the years ahead: recognition and embodiment of fundamental values into the architecture of the web.
-
-# Our Values for the world-wide web
-
-These are the core values of the Web:
+# Vision for the World-Wide Web
 
 * The Web is for all humanity.
 * The Web is designed for the good of its users.
 * The Web must be safe for its users.
 * There is one interoperable world-wide Web.
 
-# Our action as a Consortium
+# Vision for the W3C
+
+The W3C is an association where diverse voices from around the world and industries come together to incubate and build consensus for global standards that make up the Web, serving the Vision for the Web.
+
+* We put the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
+* This organization believes in diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. 
+* We strongly emphasize accessibility, internationalization, privacy, and security.
+* Our standards are rooted in a strong royalty-free patent policy and open copyright licenses, openly developed with consensus of industry and key stakeholders.
+
+# Mission of the W3C
+The fundamental purpose of the W3C is to provide an open forum, where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
 
-We will continue to develop a world-wide web that
-* Works for everyone, striving for diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. We strongly emphasize accessibility, internationalization, privacy, security and diversity.
-* Puts the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-* Is trustworthy, by ensuring security and privacy for users. 
-* Does not favor centralization.
-* Implements a unified, extensible, Web architecture, which continues to address evolving use cases for the general public.
-* Is based on standards that are openly developed with consensus of industry and key stakeholders, standards rooted in a strong royalty-free patent policy and open copyright licenses, focussed on interoperability and collective empowerment, and supported by open test suites.
+As the W3C leads the Web forward, our mission is to recognize and embody fundamental values and principles into the architecture of the web.  We must become more principled in our execution of the vision of the Web.
```

Comment:
This is partly new, in order to clearly call out that we must become more principled.

Reply:
Replicating comments from https://github.com/w3c/AB-public/pull/28#discussion_r1048819585:

@[cwilso](https://github.com/cwilso) cwilso [5 hours ago](https://github.com/w3c/AB-public/pull/28#discussion_r1048819585)
This purpose/mission section may still need work.

@[michaelchampion](https://github.com/michaelchampion) michaelchampion [5 hours ago](https://github.com/w3c/AB-public/pull/28#discussion_r1048862689)
Yup, needs work but is basically OK

@[dwsinger](https://github.com/dwsinger) dwsinger [1 hour ago](https://github.com/w3c/AB-public/pull/28#discussion_r1049049127)
Yes, the lead sentences in these two sections seem to echo each other